test/support/factory.ex

```diff
@@ -107,7 +107,10 @@ defmodule T.Factory do
       gender: gender,
       name: opts[:name] || "that",
       times_liked: opts[:times_liked] || 0,
-      filters: %{genders: opts[:accept_genders] || ["F"]}
+      gender_preference: opts[:accept_genders] || ["F"],
+      distance: opts[:distance] || 20000,
+      max_age: opts[:max_age] || 100,
+      min_age: opts[:min_age] || 18
```

Comment:
since these are nullable, we shouldn't give them defaults in factory. otherwise it's impossible to test users without these fields